bundles/net.tourbook/src/net/tourbook/importdata/EasyImportManager.java

```diff
@@ -1278,6 +1278,6 @@ private void setTourType(final TourData tourData, final ImportLauncher importLau
             String.format(//
                   LOG_EASY_IMPORT_003_TOUR_TYPE_ITEM,
                   tourData.getTourStartTime().format(TimeTools.Formatter_DateTime_S),
-                  tourTypeName + " (" + tourTypeCadence + ")"));
+                  tourTypeName + " (" + tourTypeCadence.getNlsLabel() + ")"));

```

Comment:
I do this comment in the old PR because I cannot do it in the new PR.

You should run the Eclipse tool `Externalize Strings` that all of your new strings are externalized or ignored.